vsphere/resource_vsphere_virtual_machine.go

```diff
@@ -283,6 +283,23 @@ func resourceVSphereVirtualMachine() *schema.Resource {
 			Computed:    true,
 			Description: "The power state of the virtual machine.",
 		},
+		"usb_controller": {
```

Comment:
Based on the vSphere configuration maximums for v7 and v8m USB 1.x, 2.x, and 3.x is supported; however, only one USB host controller **of each version** 1.x, 2.x, or 3.x can be added at the same time.